### ISSUE TITLE
Additions to the pi-hole plugin

### ISSUE THREAD

**Issue by C4Wiz** [matryer/bitbar-plugins#1040]

**Comment by gingerbeardman** [comment]
Yes, the process here on GitHub is that you create a fork of the project and make the edit, then send it back to us as a Pull Request. 

The easiest way to accomplish this is to browse to the file on gitHub.com, click the edit pencil button, make your edit, save and then press the pull request button. 

We'd appreciate your help on this! Thanks. 

Reply if you need any more guidance.

**Comment by C4Wiz** [comment]
Im sorry i didnt specify, i was wondering if the author could add it in his coding format. It’s much cleaner then mine

**Comment by C4Wiz** [comment]
@fmartingr 

i have expanded your pi-hole plugin, and would like to chat about capabilities

i have integrated the new api.php?versions to include:
displaying current installed versions of core, web, FTL as well as branch. and also a update check and display. i have also included the last blocked domain: please see screenshots below
<img width="281" alt="screen shot 2018-06-21 at 9 17 53 am" src="https://user-images.githubusercontent.com/3883521/41728463-1c9d2c2c-7534-11e8-9302-e92cd336ddc8.png">
<img width="280" alt="screen shot 2018-06-21 at 9 19 37 am" src="https://user-images.githubusercontent.com/3883521/41728538-45dbbe5a-7534-11e8-9162-f5261d54f8cc.png">

**Comment by fmartingr** [comment]
Hey @C4Wiz! Thanks for reaching. Keep in mind that this is not _my_ plugin, but I don't mind adding the improvements you suggested to the code if you don't want to do it.

Just notice that I no longer use Pi-hole at home so this will take some time for me.

@gingerbeardman put me on assigned if you want so I do not forget about this!

**Comment by C4Wiz** [comment]
my changes are based on the development branch that includes the new api calls for versions

i would also like to cange the enable/disable so a webpage is not opened. i has it working with urlopen, but it kept looping everytime the plugin refreshed

i would also like to add the ability to run the pihole -up command from clicking on the update available, also could not figure that one out
